pages/_error.tsx

```
@@ -0,0 +1,39 @@
+import { NextPageContext } from 'next';
+
+interface ErrorProps {
+  statusCode?: number;
+  hasGetInitialProps?: boolean;
+  err?: Error;
+}
+
+function Error({ statusCode, hasGetInitialProps, err }: ErrorProps) {
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)